### PR TITLE
Remove specs2 binds

### PR DESCRIPTION
**Pull request by liucijus**
Removes binds from specs2 test rules. Users should use testing toolchain deps to customize specs2 dependencies.

### CODE REVIEW

**Review by ittaiz — APPROVED** [review]
LGTM.
Should we have a docs section about commits with breaking changes and maybe links to the code migration suggestion?

**Comment by liucijus** [comment]
> LGTM.
> Should we have a docs section about commits with breaking changes and maybe links to the code migration suggestion?

Maybe then we should move to releases? Otherwise, it's hard to follow what happened when, and which version is following which.

**Comment by ittaiz** [comment]
We can. I feel releases might be a bit too big of a hammer for this need but I'm open to it.
Also I'm not sure I'd like to do it here and now.
I think we should discuss this in slack and get some feedback. Or maybe just link to a github issue about it.
Concretely do you think we should merge as is without documenting this change?
Also cc @simuons

**Comment by liucijus** [comment]
> We can. I feel releases might be a bit too big of a hammer for this need but I'm open to it.
> Also I'm not sure I'd like to do it here and now.
> I think we should discuss this in slack and get some feedback. Or maybe just link to a github issue about it.
> Concretely do you think we should merge as is without documenting this change?

I think this change is compatible with existing [docs](https://github.com/bazelbuild/rules_scala/blob/master/docs/testing.md), probably it's ok to merge. It should not be hard to migrate to toolchains.
